client/src/components/Spotlight.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import logoImage from './../images'
+import './css/Spotlight.css';
+
+/**
+ * Spotlight/Impact component.
+ */
+class Spotlight extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+          currentTab: 0
+        }
+        this.onClickItem = this.onClickItem.bind(this);
+        this.setTab = this.setTab.bind(this);
+    }
+
+    onClickItem(position) {
+        // TODO: Show detailed information about chosen icon
+        alert("See more information about feature " + position);
+    }
+
+    setTab(tab) {
+      this.setState({currentTab: tab});
+    }
+
+    render() {
+        const { currentTab } = this.state;
+
+        var spotlightStyle, impactStyle, spotlightContainer, impactContainer = "hidden";
+        if (currentTab === 0) {
+          spotlightStyle = "spotlight-underline";
```

Comment:
This looks like stuff you could put in a method, and the variables can be put in state vars.

Reply:
Done!